src/ui/starter-select-ui-handler.ts

```diff
@@ -989,16 +1017,20 @@ export default class StarterSelectUiHandler extends MessageUiHandler {
       return;
     }
 
+    const passiveUnlocked = this.isPassiveUnlocked(species.speciesId);
```

Comment:
```suggestion
    const isPassiveUnlocked = this.isPassiveUnlocked(species.speciesId);
```
`is` (or `has`) prefix helps to quickly distinguish a boolean